pydle/features/rfc1459/client.py

```diff
@@ -547,7 +529,7 @@ async def on_raw_pong(self, message):
     async def on_raw_invite(self, message):
         """ INVITE command. """
         nick, metadata = self._parse_user(message.source)
-        self._sync_user(nick, metadata)
```

Comment:
Can we please have functional changes extracted to a seperate PR, to make the changes easier to track in post